build.sbt

```diff
@@ -796,6 +791,7 @@ lazy val hasLargeIntegrationTests = Seq(
 )
 
 lazy val nativeSettings = Seq(
+  SettingKey[Boolean]("ide-skip-project") := true,
```

Comment:
Wasn't this obviated by one of the recent fixes to Intellij?

Reply:
It was, but this settings speeds up IntelliJ import regardless since it avoids exporting/indexing native projects. Should probably do the same for JS sources.